specs/cli/manifest-add.md

```diff
@@ -0,0 +1,163 @@
+# kiota manifest add
```

Comment:
The more I think about this, the more convicted I am that this is not the right name for the thing that the developer is creating.  Developers don't intentionally create manifests.  Manifests are an artifact of creating some other software component.  I think we are providing the ability for developers to use Kiota to generate plugins.  That is what they are trying to achieve.  The manifest is just an implementation detail.  If in the future we move to a world where plugins are defined by augmented OpenAPI descriptions, there won't even be a manifest.

Reply:
Naming was updated to "Plugin"